pkg/tms/tms.go

```diff
@@ -95,6 +95,10 @@ type CommunicationInterface interface {
 	UploadFileToNode(nodeName, fileId, description, namedUser string) (NodeUploadResponseEntity, error)
 }
 
+const (
+	DEFAULT_TR_DESCRIPTION = "Created by Piper"
```

Comment:
For one constant this can be one line

Reply:
yes, but the syntax is correct. Must it be changed to one line or it's just a proposal, like nice to have?